src/portal/src/Portal.js

```diff
@@ -1,5 +1,5 @@
 import { Component } from 'react'
-import canUseDom from 'dom-helpers/util/inDOM'
+import canUseDom from 'dom-helpers'
```

Comment:
Does this need to be `import { canUseDOM } from 'dom-helpers'`?

Honestly it seems silly to import this lib just for checking if the window object is not undefined, especially since we have that check in a couple other places too, internally.

Reply:
Hi Matt,

Thanks for looking at this, I'm not a javascript guy by trade so forgive me if I'm a little slow, `dom-helpers` [uses export default](https://github.com/react-bootstrap/dom-helpers/blob/995f5aff37f7e7baedc75f530d8ceb11c5ff9893/src/canUseDOM.ts#L1) for each module hence the default import syntax.

I'm easy on the direction you'd prefer here; it looks like `canUseDom` is potentially being more cautious by checking for `createElement` and additionally constrains the check to one place.

```javascript
export default !!(
  typeof window !== 'undefined' &&
  window.document &&
  window.document.createElement
)
```

Regardless, I'll happily follow your lead. I'll make other usages consistent if that's suitable as well.

Reply:
The problem with the latest version is that the default export of the library is not `canUseDOM` but rather a suite of functions:
https://github.com/react-bootstrap/dom-helpers/blob/995f5aff37f7e7baedc75f530d8ceb11c5ff9893/src/index.ts#L69-L101

Reply:
Aha, yes I've definitely got the wrong end of at least one stick.

However, this syntax `import { canUseDom } from 'dom-helpers'` yields this error on build:

     Error: 'canUseDom' is not exported by node_modules/dom-helpers/esm/index.js, imported by src/portal/src/Portal.js

Although this is fine `import canUseDom from 'dom-helpers/canUseDom'`.

Reply:
Aha, yes, it seems it is because it's not actually being exported by the library directly! 😅 we'll have to reference the path like you've done in your last comment: `import canUseDom from 'dom-helpers/canUseDom'` 

I'm also not opposed to doing:
```sh
yarn remove dom-helpers
```

And then instead of pulling in this whole lib just for a simple function we can have this module internally:
```js
export default function canUseDom() {
  return typeof window !== 'undefined' && window.document
}
```
Then anywhere we do `typeof window !== 'undefined'` or `typeof document !== 'undefined'` we can replace it with this function call. Wdyt?

Reply:
That sounds good to me, I'll do it asap, thanks Matt!